docs/contributor/02-30-keb-configuration.md

```diff
@@ -1,36 +1,31 @@
+| **APP_PORT**                                                   | Specifies the port on which the HTTP server listens.                                                                         |                `8080`                |
+| **APP_DATABASE_USER**                                          | Defines the database username.                                                                                               |              `postgres`              |
+| **APP_DATABASE_PASSWORD**                                      | Defines the database user password.                                                                                          |              `password`              |
+| **APP_DATABASE_HOST**                                          | Defines the database host.                                                                                                   |             `localhost`              |
+| **APP_DATABASE_PORT**                                          | Defines the database port.                                                                                                   |                `5432`                |
+| **APP_DATABASE_NAME**                                          | Defines the database name.                                                                                                   |               `broker`               |
+| **APP_DATABASE_SSLMODE**                                       | Specifies the SSL Mode for PostgreSQL. See [all the possible values](https://www.postgresql.org/docs/9.1/libpq-ssl.html).    |              `disable`               |
+| **APP_DATABASE_SSLROOTCERT**                                   | Specifies the location of CA cert of PostgreSQL. (Optional)                                                                  |                 None                 |
+| **APP_TRIAL_REGION_MAPPING_FILE_PATH**                         | Defines a path to the file which contains a mapping between the platform region and the trial plan region.                   |                 None                 |
+| **APP_GARDENER_PROJECT**                                       | Defines the project in which the cluster is created.                                                                         |              `kyma-dev`              |
+| **APP_GARDENER_SHOOT_DOMAIN**                                  | Defines the domain for clusters created in Gardener.                                                                         | `shoot.canary.k8s-hana.ondemand.com` |
+| **APP_GARDENER_KUBECONFIG_PATH**                               | Defines the path to the kubeconfig file for Gardener.                                                                        |  `/gardener/kubeconfig/kubeconfig`   |
```

Comment:
The default value is not rendered correctly in the preview. I assume the line's too long.
![Screenshot 2025-05-07 at 10 10 38](https://github.com/user-attachments/assets/f7d85c11-e003-49b0-b1ce-f6f674fc7b5d)

Reply:
The previous line is longer, so I am confused.
![image](https://github.com/user-attachments/assets/adfd64d6-070b-4c59-bb96-41a8bd88173c)

Reply:
This is how I see it in the preview:
![Screenshot 2025-05-07 at 11 05 18](https://github.com/user-attachments/assets/b46667b5-acaa-402a-89da-464f89b91c14)

Reply:
My mistake, sorry. LGTM, so please ignore the comment.